packages/react-component-library/src/components/Autocomplete/AutocompleteOption.tsx

```diff
@@ -14,7 +14,7 @@ export const AutocompleteOption = React.forwardRef<
 >(({ children, inputValue, ...rest }, ref) => {
   return (
     <SelectBaseOption ref={ref} {...rest}>
-      {reactStringReplace(children, inputValue, (match, i) => (
+      {reactStringReplace(children, inputValue || undefined, (match, i) => (
```

Comment:
It seems to match options containing the text 'Undefined' like this unfortunately. I think we probably need to skip `reactStringReplace` completely if `inputValue` is falsey.

Reply:
👍